ui/org.eclipse.pde.ui.templates/templates_3.1/view/java/$className$.java

```diff
@@ -217,7 +217,7 @@ public void createPartControl(Composite parent) {
 		
 %if viewType == "tableViewer"
 		viewer.setContentProvider(ArrayContentProvider.getInstance());
-		viewer.setInput(new String[] { "One", "Two", "Three" });
+		viewer.setInput(List.of("One", "Two", "Three"));
```

Comment:
Why is this an improvement?
At runtime this will cause an extra copy of the list into an array.